### PR TITLE
Bugfix/electron 11

### PR DESCRIPTION
**Pull request by rssk**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our [guidelines](../CONTRIBUTING.md)
- [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
- [] Tests for the changes have been added (for bug fixes / features)
- [] Passes e2e testing
- [x] Passes lint
- [] Docs have been added / updated if necessary (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Bugfix for the electron upgrade, as well as a bunch of package updates

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
Very yes, a bunch of breaking change going to this version of electron v11 + hapi v20

### CODE REVIEW

File: packages/coinstac-api-server/src/index.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ async function startServer() {
     },
     subscriptions: {
       async onConnect(connectionParams, ws) {
+        debugger
```
Comment (dylanmartin) [inline packages/coinstac-api-server/src/index.js]: Is this here on purpose?

File: packages/coinstac-api-server/src/auth-helpers.js [hunk 2]
```diff
@@ -10,14 +10,24 @@ const { eventEmitter, USER_CHANGED } = require('./data/events');
 
 sgMail.setApiKey(process.env.SENDGRID_API_KEY);
 
+const audience = 'coinstac';
```
Comment (rochaeb) [inline packages/coinstac-api-server/src/auth-helpers.js:13]: maybe set this via env variables?

File: packages/coinstac-api-server/src/index.js (hunk 1 above)
Comment (rssk) [reply packages/coinstac-api-server/src/index.js]: nah just from when I pushed for Ed, removed!

File: packages/coinstac-ui/app/main/utils/boot/configure-browser-window.js [hunk 3]
```diff
@@ -74,7 +76,7 @@ function createWindow() {
           protocol: 'file:',
           slashes: true,
           webPreferences: {
-            devTools: false,
+            devTools: true,
```
Comment (rochaeb) [inline packages/coinstac-ui/app/main/utils/boot/configure-browser-window.js]: could set to false when in production

**Review by rochaeb — APPROVED** [review]

File: packages/coinstac-ui/app/main/utils/boot/configure-browser-window.js (hunk 3 above)
Comment (rssk) [reply packages/coinstac-ui/app/main/utils/boot/configure-browser-window.js]: yeah my bad this was debugging a test issue

File: packages/coinstac-api-server/src/auth-helpers.js (hunk 2 above)
Comment (rssk) [reply packages/coinstac-api-server/src/auth-helpers.js:13]: agreed, but we need to figure out what the best practice is here, I think this is just to get this off the ground for now

**Pull request merged**